test/NumSharp.UnitTest/Extensions/NdArray.VStack.Test.cs

```diff
@@ -29,19 +29,19 @@ public void VStackNDArrays()
             Assert.IsTrue(n[1, 2] == 4);
 
             //2D
-            n1 = np.array(new double[][] { new double[] { 1 }, new double[] { 2 }, new double[] { 3 } });
-            n2 = np.array(new double[][] { new double[] { 4 }, new double[] { 5 }, new double[] { 6 } });
+            var n01 = np.array(new double[][] { new double[] { 1 }, new double[] { 2 }, new double[] { 3 } });
```

Comment:
This renaming of variables should not have been necessary. However, fixing the Shape.size caused this test to fail because the reassignment itself has a bug that I did not have time to chase down.